README.md

```diff
@@ -30,64 +30,78 @@ We suggest a couple of alternative ways of installation.
 
 git-machete works under both Python 2.7 and Python 3.x.
 
-### Using Homebrew (macOS)
+<details><summary><b>Using Homebrew (macOS)</b></summary><br/>
```

Comment:
Theoretically, spoilers can be replaced by headers because there is a little content inside it, but IMHO spoilers are also OK